enterprise/metrics/src/main/java/org/neo4j/metrics/source/ClusterMetrics.java

```diff
@@ -86,6 +124,12 @@ public void stop() throws IOException
             registry.remove( SLAVE_PULL_UPDATES );
             registry.remove( SLAVE_PULL_UPDATE_UP_TO_TX );
 
+            if( clusterMembers != null )
```

Comment:
this condition should be moved up to the other if, i.e., ` if ( config.get( MetricsSettings.neoClusterEnabled ) && clusterMembers != null )`